lib/keycloak.js

```diff
@@ -299,7 +297,7 @@ function Keycloak (config) {
                         });
                     });
                 } else {
-                    kc.updateToken(-1).then(function() {
+                    kc.updateToken(0).then(function() {
```

Comment:
Let's omit the `minValidity` argument here, as it will default to `5`  when `0` is passed due to the following code:

https://github.com/keycloak/keycloak-js/blob/7f5df13236b487d92a44f0155aacf6a6cea3ee5f/lib/keycloak.js#L629